### PR TITLE
Adaptive for banner and partners

### PR DESCRIPTION
**Pull request by vvbogdanovih**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/features/MainPage/StaticBanners/StaticBanner.styles.scss [hunk 1]
```diff
@@ -67,4 +71,303 @@
...
+
+                .redirectButton {
+                    @include mut.rem-margined($left: 0px);
+                    @include mut.rem-padded($left: 10px);
+                    @include mut.sized($width: 180px, $height: 46px);
+                    @include mut.full-rounded(10px);
+                    @include mut.with-font($font-family: ft.$roboto-font, $font-weight: 500, $font-size: 14px);
+                    @include mut.flexed($direction: center, $justify-content: center, $align-items: center);
+                    min-width: f.pxToRem(180px);
+                    line-height: 16.41px;
```
Comment (NadiaKishchuk) [inline src/features/MainPage/StaticBanners/StaticBanner.styles.scss:206]: you have a lot of duplicated styles in different screen extensions, please fix it

**Review by NadiaKishchuk — CHANGES_REQUESTED** [review]

File: src/features/MainPage/StaticBanners/StaticBanner.styles.scss (hunk 1 above)
Comment (vvbogdanovih) [reply src/features/MainPage/StaticBanners/StaticBanner.styles.scss]: Removed styles that are repeated and do not affect the display of the page

File: src/features/MainPage/PartnersBlockMain/PartnersBlockMain.styles.scss [hunk 2]
```diff
@@ -3,17 +3,112 @@
...
         max-width: 1920px;
         margin: 0 auto;
+
+        .keyPartnersBlock {
+            column-gap: 0;
+            row-gap: 0  ;
+
+            .partnersItem {
+                @include mut.rem-margined($top: 30px, $right: 10px, $bottom: 30px, $left: 10px);
+
```
Comment (MaksBrat) [inline src/features/MainPage/PartnersBlockMain/PartnersBlockMain.styles.scss:17]: pls remove the blank line

File: src/features/MainPage/PartnersBlockMain/PartnersBlockMain.styles.scss [hunk 3]
```diff
@@ -3,17 +3,112 @@
...
+@media screen and (max-width: 1200px) {
+    .partnersBlock {
+        .keyPartnersBlock {
+            .partnersItem {
+                @include mut.rem-margined($top: 15px, $right: 10px, $bottom: 10px, $left: 15px);
+                @include mut.sized($width: 350px, $height: 106px);
+            }
+        }
+    }
+
```
Comment (MaksBrat) [inline src/features/MainPage/PartnersBlockMain/PartnersBlockMain.styles.scss:40]: same)

File: src/features/MainPage/PartnersBlockMain/PartnersBlockMain.styles.scss [hunk 4]
```diff
@@ -3,17 +3,112 @@
...
+}
+
+@media screen and (max-width: 768px) {
+    .partnersBlock {
+        .keyPartnersBlock {
+            .partnersItem {
+                @include mut.rem-margined($top: 20px, $right: 15px, $bottom: 20px, $left: 15px);
+                @include mut.sized($width: 340px, $height: 98px);
+
```
Comment (MaksBrat) [inline src/features/MainPage/PartnersBlockMain/PartnersBlockMain.styles.scss:49]: here and in the following blocks as well

**Review by MaksBrat — CHANGES_REQUESTED** [review]

File: src/features/MainPage/PartnersBlockMain/PartnersBlockMain.styles.scss (hunk 4 above)
Comment (vvbogdanovih) [reply src/features/MainPage/PartnersBlockMain/PartnersBlockMain.styles.scss]: Removed blank lines

File: src/features/MainPage/PartnersBlockMain/PartnersBlockMain.styles.scss [hunk 5]
```diff
@@ -3,17 +3,103 @@
...
+            }
+        }
+    }
+}
+
+@media screen and (max-width: 620px) {
+    .partnersBlock {
+        .keyPartnersBlock {
+            .partnersItem {
+                @include mut.rem-margined($top: 15px, $right: 12px, $bottom: 15px, $left: 12px);
```
Comment (MaksBrat) [inline src/features/MainPage/PartnersBlockMain/PartnersBlockMain.styles.scss:78]: duplication 67 and 78 lines

File: src/features/MainPage/PartnersBlockMain/PartnersBlockMain.styles.scss [hunk 6]
```diff
@@ -3,17 +3,103 @@
...
+            }
+        }
+    }
+}
+
+@media screen and (max-width: 768px) {
+    .partnersBlock {
+        .keyPartnersBlock {
+            .partnersItem {
+                @include mut.rem-margined($top: 20px, $right: 15px, $bottom: 20px, $left: 15px);
```
Comment (MaksBrat) [inline src/features/MainPage/PartnersBlockMain/PartnersBlockMain.styles.scss:45]: duplication 45 and 56 lines

Comment (vvbogdanovih) [reply src/features/MainPage/PartnersBlockMain/PartnersBlockMain.styles.scss:45]: Removed duplicates in PartnersBlockMain.styles

**Review by MaksBrat — APPROVED** [review]

File: src/features/MainPage/PartnersBlockMain/PartnersBlockMain.styles.scss [hunk 7]
```diff
@@ -3,17 +3,101 @@
...
+
+@media screen and (max-width: 382px) {
+    .partnersBlock {
+        .keyPartnersBlock {
+            .partnersItem {
+                @include mut.rem-margined($top: 10px, $right: 10px, $bottom: 10px, $left: 10px);
+                @include mut.sized($width: 160px, $height: 46px);
+            }
+        }
+    }
```
Comment (NadiaKishchuk) [inline src/features/MainPage/PartnersBlockMain/PartnersBlockMain.styles.scss:102]: I think it is better to use margin auto for left and right if possible.  So you must create the wrapper div and the element must be inside. it looks like it is now in the same way.

File: src/features/MainPage/StaticBanners/StaticBanner.styles.scss [hunk 8]
```diff
@@ -67,4 +71,285 @@
...
+
+        .left,
+        .right {
+            display: none;
+        }
+
+        &.catalog {
+            @include mut.flexed($direction: column, $justify-content: center, $align-items: center);
+
+            .mainContainer {
```
Comment (NadiaKishchuk) [inline src/features/MainPage/StaticBanners/StaticBanner.styles.scss:116]: in your component, you don`t have the catalog class used.  So why it is here?

**Review by Dobriyr — CHANGES_REQUESTED** [review]
image
Cutting border-radius when screen width  lover then 350px
it seems to me that in the styles you should change the width to 100% instead of fixed
---
There is also no box-shadow
Your button
image

button on Figma
image

**Review by NadiaKishchuk — APPROVED** [review]

**Review by KateYatsiuk — APPROVED** [review]

**Review by Dobriyr — APPROVED** [review]

**Pull request merged**